homeassistant/components/roborock/image.py

```diff
@@ -66,10 +67,22 @@ def __init__(
         self.map_flag = map_flag
         self.cached_map = self._create_image(starting_map)
 
+    @property
+    def entity_category(self) -> EntityCategory | None:
+        """Return diagnostic entity category for any non-selected maps."""
+        if not self.is_selected:
```

Comment:
I'm not entirely sure that I think that the selected image is less diagnostic information than the non selected images or vice versa. But I can't completely dismiss it either, so I'll approve.